app/uk/gov/hmrc/merchandiseinbaggage/model/api/CalculationRequest.scala

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ */
+
+package uk.gov.hmrc.merchandiseinbaggage.model.api
+
+import play.api.libs.json.{Format, Json}
+import uk.gov.hmrc.merchandiseinbaggage.model.core.AmountInPence
+
+//TODO currency is just a string for now - waiting for conversion integration
+case class CalculationRequest(currency: String, amount: AmountInPence)
```

Comment:
AmountInPence shouldn't be used for any currency other than sterling, whereas if I've read it correctly this method takes an amount in any currency. Maybe just a Long or BigDecimal, else a CurrencyAmount?